kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -285,7 +285,7 @@ func (c *ipamController) acceptScheduleRequests(stopCh <-chan struct{}) {
 			err := c.syncIPAM()
 			if err != nil {
 				// We can kick ourselves on error for a retry. We have rate limiting
-				// built in to the cleanup code.
+				// built into the cleanup code.
```

Comment:
changes for `into` have been fairly controversial -- happy to drop this or anything else...

Reply:
I think this one works.